app/services/ember-devtools.js

```diff
@@ -40,11 +43,28 @@ export default Ember.Object.extend({
   routes: function() {
     return Ember.keys(this.router().recognizer.names);
   },
-  view: function(id) {
-    if (typeof id === 'object') {
-      id = id.id;
+  view: function(idOrSelector) {
+    if (typeof idOrSelector === 'object') {
+      idOrSelector = idOrSelector.id;
+    }
+    return Ember.View.views[idOrSelector] || this.views(idOrSelector)[0];
+  },
+  views: function (selectorOrName) {
+    var views = Ember.View.views,
+      viewClass =  this.lookupFactory('component:' + selectorOrName) || this.lookupFactory('view:' + selectorOrName);
```

Comment:
It's a single var, just an or. I changed it to use multiple vars for consistency with line 117